config/sync/metatag.metatag_defaults.node__news.yml

```diff
@@ -0,0 +1,18 @@
+uuid: 00762435-258d-4e52-bd0f-09e5ca2771f5
+langcode: en
+status: true
+dependencies: {  }
+id: node__news
+label: 'Content: News'
+tags:
+  schema_article_author: 'a:3:{s:5:"@type";s:6:"Person";s:4:"name";s:26:"[node:author:display-name]";s:3:"url";s:17:"[node:author:url]";}'
+  schema_article_type: NewsArticle
+  schema_article_main_entity_of_page: '[current-page:url]'
+  schema_article_name: '[node:title]'
+  schema_article_about: '[node:field_keywords]'
+  schema_article_date_published: '[node:published]'
+  schema_article_headline: '[node:field_news_headline]'
+  schema_article_date_modified: '[node:changed]'
+  schema_article_publisher: 'a:3:{s:5:"@type";s:12:"Organization";s:4:"name";s:11:"[site:name]";s:3:"url";s:10:"[site:url]";}'
+  schema_article_description: '[node:body]'
+  schema_article_image: 'a:5:{s:5:"@type";s:11:"ImageObject";s:20:"representativeOfPage";s:4:"True";s:3:"url";s:26:"[node:field_news_logo:entity:url]";s:5:"width";s:28:"[node:field_news_logo:width]";s:6:"height";s:29:"[node:field_news_logo:height]";}'
```

Comment:
This link was part of the ticket description :P I am going to recheck this.

Reply:
I changed this to test that summary is preferred (after I fixed it). It turns out that it is default token and metatag functionality that if you use `[node:summary]` as a token it will use the summary of the body and if this is blank, the trimmed body. I tested it and it trims it to 597 characters.